modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -81,6 +95,12 @@ From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemite
 From the {webui}, add [systemitem]``RHEL7 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
 +
 From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
+* For {es} 8:
++
+From the {webui}, add [systemitem]``RHEL8 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES8 x86_64``.
+You will also need to add the ``Appstream`` channel.
```

Comment:
BTW: Why is this here? We don't mention that you need `os` channel for the other versions (and neither to mention the user needs `BaseOS` for SLES ES 8.

I think this can go, I we should focus on `.Procedure: Adding Base Media to Custom Channels`, that is still missing information for SLES ES 8.